src/vendor/bullet.js

```diff
@@ -8,6 +8,7 @@ export default function bullet () {
     markers = bulletMarkers,
     measures = bulletMeasures,
     measureHeight = bulletMeasureHeight,
+    rangeMaxx = getRangeMax,
```

Comment:
isn't getRangeMax better than a variable with an intentional typo? or is the second x for x-axis? in that rangeMaxX would make that a bit clearer